packages/rocketchat-lib/server/functions/saveUser.js

```diff
@@ -94,6 +94,48 @@ function validateUserData(userId, userData) {
 	}
 }
 
+function validateUserEditing(userId, userData) {
+	const editingMyself = userData._id && userId !== userData._id;
+
+	const canEditOtherUserInfo = RocketChat.authz.hasPermission(userId, 'edit-other-user-info');
+	const canEditOtherUserPassword = RocketChat.authz.hasPermission(userId, 'edit-other-user-password');
+
+	if (!RocketChat.settings.get('Accounts_AllowUserProfileChange') && !canEditOtherUserInfo && !canEditOtherUserPassword) {
```

Comment:
This will cause the permissions to override the setting. Is that the desired behavior?

Reply:
that's right.. an admin should be able to edit other's user info.